knplabs/knp-snappy/CVE-2023-28115.yaml

```diff
@@ -0,0 +1,8 @@
+title:     PHAR deserialization allowing remote code execution
+link:      https://github.com/advisories/GHSA-gq6w-q6wh-jggc
+cve:       CVE-2023-28115
+branches:
+    1.x:
+        time:     2023-03-17 15:47:00
+        versions: ['<=1.4.1']
```

Comment:
We generally write constraints based on the version including the patch, so `<1.4.2`